src/components/slack/list/slacklist.component.ts

```diff
@@ -269,4 +286,40 @@ export class SlackListComponent implements OnInit, OnDestroy {
     onClickSetting() {
         this.router.navigate(['setting']);
     }
+
+    onSelectChannelRequest(id: string) {
+        let teamID = id.split('-')[0];
+        let channelID = id.split('-')[1];
```

Comment:
Can you use 'const' (instead of 'let') for variables that are never changed?

Reply:
fixed